chia/full_node/mempool_manager.py

```diff
@@ -324,23 +277,25 @@ async def pre_validate_spendbundle(
 
         self._worker_queue_size += 1
         try:
-            err, cached_result_bytes, new_cache_entries, duration = await asyncio.get_running_loop().run_in_executor(
+            sbc, new_cache_entries, duration = await asyncio.get_running_loop().run_in_executor(
                 self.pool,
                 validate_clvm_and_signature,
-                new_spend_bytes,
+                new_spend,
                 self.max_tx_clvm_cost,
                 self.constants,
                 self.peak.height,
             )
+        except Exception as e:  # take returned TypeError and turn into ValidationError class by
+            error = Err(e.args[0])
```

Comment:
how do we know that `args[0]` exists?
It seems safer to catch the specific exception type that we expect, and pull out the error code where we expect it to be. If we get an unexpected exception, we should probably just throw `ValidationError` with some more generic error code.